docs/getting-started-guides/locally.md

```diff
@@ -129,6 +132,19 @@ cluster/kubectl.sh create -f docs/user-guide/pod.yaml
 
 Congratulations!
 
+### Hacking hack-local-up cluster to test more sophisticated deployments, such as HA.
+
+To test highly available apiservers, you can modify local-up-cluster.sh as follows.
+
+1) Write a kubernetes token file which has a kubelet user with a token defined.  Lets call it /tmp/mytokens for concreteness.
+
+2) Launch the apiserver with the `--token-auth-file=/tmp/mytokens` and `--apiserver-node-port=30279` options.  Of course 30279 is just an example port.
```

Comment:
even if the cert works, why use teh node-port internally?

On Wed, Sep 23, 2015 at 4:58 PM, Jordan Liggitt <notifications@github.com>
wrote:

> In docs/getting-started-guides/locally.md
> <https://github.com/kubernetes/kubernetes/pull/13978#discussion_r40273179>
> :
>
> > @@ -129,6 +132,19 @@ cluster/kubectl.sh create -f docs/user-guide/pod.yaml
> >
> >  Congratulations!
> >
> > +### Hacking hack-local-up cluster to test more sophisticated deployments, such as HA.
> > +
> > +To test highly available apiservers, you can modify local-up-cluster.sh as follows.
> > +
> > +1) Write a kubernetes token file which has a kubelet user with a token defined.  Lets call it /tmp/mytokens for concreteness.
> > +
> > +2) Launch the apiserver with the `--token-auth-file=/tmp/mytokens` and `--apiserver-node-port=30279` options.  Of course 30279 is just an example port.
>
> Right, the serving cert. I brought this point up before. @jayunit100
> <https://github.com/jayunit100>, were you able to test this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13978/files#r40273179>.
>